pages/deployments/oidc/oidc_with_aws.md

```diff
@@ -0,0 +1,168 @@
+---
+keywords: oidc, authentication, IAM, roles, AWS
+---
+
+# OIDC with AWS
+
+The [Buildkite Agent's oidc command](/docs/agent/v3/cli-oidc) allows you to request an OIDC token representing the current job. These tokens can be exchanged with federated systems like AWS in exchange for an IAM role with AWS scoped permissions.
```

Comment:
Need to initially spell out what IAM is (i.e. "Identity and Access Management (IAM)").